src/instrumentation/emitter.js

```diff
@@ -16,19 +18,35 @@ module.exports = class InstrumentationEventEmitter {
       throw new KafkaJSError('Invalid event name', { retriable: false })
     }
 
-    if (this.emitter.listenerCount(eventName) > 0) {
+    if (this.forwarders.length > 0 || this.emitter.listenerCount(eventName) > 0) {
       const event = new InstrumentationEvent(eventName, payload)
       this.emitter.emit(eventName, event)
+      this.forwarders.forEach(forwarder => forwarder.emitForwarded(eventName, event))
     }
   }
 
+  /**
+   * @private
+   */
+  emitForwarded(eventName, event) {
+    this.emitter.emit(eventName, event)
+  }
+
   /**
    * @param {string} eventName
-   * @param {Function} listener
-   * @returns {Function} removeListener
+   * @param {(...args: any[]) => void} listener
+   * @returns {import("../../types").RemoveInstrumentationEventListener<string>} removeListener
    */
   addListener(eventName, listener) {
     this.emitter.addListener(eventName, listener)
     return () => this.emitter.removeListener(eventName, listener)
   }
+
+  forward(anotherInstrumentationEmitter) {
```

Comment:
Is it intentional here that you can subscribe the same forwarder more than once, or should `this.forwarders` perhaps be a set?